pyoozie/__init__.py

```diff
@@ -38,7 +38,7 @@
 from pyoozie.tags import WorkflowApp
 
 
-__version__ = '0.0.1'
+__version__ = '0.0.2'
```

Comment:
Normally we create another PR to bump the version (b/c there's usually a backlog of changes to release), but since there's no backlog, we can bump in this PR 👍